service/status-report/main.go

```diff
@@ -138,7 +145,9 @@ func main() {
 
 	status := providerclient.NewStorageClientStatus().
 		SetPlatformVersion(pltVersion).
-		SetOperatorVersion(oprVersion)
+		SetOperatorVersion(oprVersion).
+		SetClusterID(string(clusterID)).
```

Comment:
why are we sending the clusterID again? i believe it should be available on the storageConsumer CR. isn't that the case?

Can't we send this information when onboarding not each time during the status report?

Reply:
See [discussion](https://github.com/red-hat-storage/ocs-operator/pull/2504#discussion_r1524166253)

Reply:
> See discussions [(2)](https://github.com/red-hat-storage/ocs-operator/pull/2504#discussion_r1524179758)

Reply:
Resolving it as got the answer